Triangulation_2/include/CGAL/Constrained_triangulation_2.h

```diff
@@ -1668,7 +1768,7 @@ intersection(const Gt& gt,
 #endif
   typedef typename Gt::Construct_bbox_2 Construct_bbox_2;
   Construct_bbox_2 bbox = gt.construct_bbox_2_object();
-  typename boost::result_of<const Construct_bbox_2(const typename Gt::Point_2&)>::type bb(bbox(pi));
+  auto bb(bbox(pi));
```

Comment:
For the projection traits that must be `Bbox_3`.

Reply:
I understand but thinking about it the projection traits should return a 2D bbox (because of the concept)

Reply:
That is what it does ( [link ](https://github.com/CGAL/cgal/blob/master/Kernel_23/include/CGAL/Kernel_23/internal/Projection_traits_3.h#L89))

Reply:
> I understand but thinking about it the projection traits should return a 2D bbox (because of the concept)

You are right. I do not know why `Bbox_2` was not hard-coded. The change was introduced by that commit: https://github.com/CGAL/cgal/commit/1123afb70e8d75f3107dc97b536959ae60c39426#diff-82a99d8ff284c2f736bdb0349548579f80f159802f9279f59434a4462dc3e305R1462.